bin/cli.js

```diff
@@ -3,12 +3,14 @@
 'use strict'
 
 const yargs = require('yargs')
-const main = require('../index.js')
+const Installer = require('../index.js')
 
-main(parseArgs()).then((details) => {
-  console.log(`added ${details.count} packages in ${
-    details.time / 1000
+new Installer(parseArgs()).then(details => {
```

Comment:
Should this be `.run().then(...)`? Does this mean the tests don't currently exercise the CLI portion?

Reply:
Correct!

Reply:
oh yeah. I fixed this in a different branch. Fix't